src/TorSharp/Tools/Tor/TorControlClient.cs

```diff
@@ -71,5 +82,24 @@ private async Task<string> SendCommandAsync(string command, string expectedRespo
 
             return response;
         }
+
+        private async Task<string> SendTrafficCommandAsync(string command, string expectedResponse)
+        {
+            if (_tcpClient == null)
+            {
+                throw new TorControlException("The Tor control client has not connected.");
+            }
+
+            await _writer.WriteLineAsync(command).ConfigureAwait(false);
+            await _writer.FlushAsync().ConfigureAwait(false);
+
+            var response = await _reader.ReadLineAsync().ConfigureAwait(false);
```

Comment:
This should read two lines (or N + 1 where N is the number of `GETINFO` names) and the last should be `250 OK`. I'm worried that if we don't read all of the lines we're not consuming all of the data that the control server is expecting to send which may lead to unexpected behavior. From the control specification is seems like we should make sure that `250 OK` is returned at the end to make sure we're in an expected state, i.e. the control server is indicating that everything went as expected fetching the request data.

Reply:
Having last line 250 OK would be better but longer to develop.
If Authenticate is OK the call command should works fine because tor in local.
I'm using it in this way, writing to an file each hour the traffic values and is working without any issue. 
Maybe you can implement it later

Reply:
Okay, sounds good!